src/main/java/net/pms/PMS.java

```diff
@@ -366,7 +355,11 @@ private void displayBanner() throws IOException {
 		 * usually done by the installer
 		 */
 		File dDir = new File(configuration.getDataDir());
-		dDir.mkdirs();
+		if (!dDir.exists()) {
```

Comment:
@SubJunk There's a reason but that has nothing to do with the final result and everything to do with how I ended up with that solution :wink: 

The "requirement" from FindBugs was that the value from .mkdirs wasn't ignored, so I added a log message if it didn't work. I then testen and found out that it "failed" if the folders already existed, and thus I wrapped it with the first ```if```. 

I'll fix :innocent: